src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/InterchangeHeaderTest.java

```diff
@@ -1,140 +1,96 @@
 package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
 
+import org.junit.Ignore;
```

Comment:
JUnit5 won't ignore tests annotated with this - it's from JUnit 4. The ignored tests are passing anyway, so probably the annotation can just be removed?

The JUnit 5 equivalent is `@Disabled`